numpy/_core/src/umath/loops_trigonometric.dispatch.c.src

```diff
@@ -39,8 +40,10 @@ simd_range_reduction_@sfx@(npyv_@sfx@ x, npyv_@sfx@ y, npyv_@sfx@ c1, npyv_@sfx@
 }
 #endif
 /**end repeat**/
-
-#if NPY_SIMD_F64
+/* Disable SIMD code and revert to libm: see
+ * https://mail.python.org/archives/list/numpy-discussion@python.org/thread/C6EYZZSR4EWGVKHAZXLE7IBILRMNVK7L/
+ * for detailed discussion on this*/
+#if 0 // NPY_SIMD_F64
```

Comment:
This is a functional change, right, not just fixing a build warning - the disabling wasn't done earlier after the May'23 mailing list discussion? If so, it'd be good to add this to the PR title and description.

Reply:
Could we just merge this and do a follow up to remove the F64 code as a distinct change? It feels like we're just cluttering the code base with it at this point given it'll need changing to 1ULP *or* adding the accuracy toggle.

Reply:
Also, not a functional change, it was disabled here first:

https://github.com/numpy/numpy/blob/6e3b92339a912f5326ff434a6f32e015c27369db/numpy/_core/src/umath/loops_trigonometric.dispatch.c.src#L391-L395

This function was just left out, causing the unused warning.

Reply:
Okay, thanks for checking! Wasn't clear to me from only the diff. Then I agree, let's get this in as is.

And +1 for figuring out whether or not to add the accuracy toggle, and otherwise removing the code.